CHANGES.rst

```diff
@@ -35,6 +35,8 @@ Improvements
 - Add an option to send notifications of new abstract comments (:issue:`5266`, :pr:`5284`)
 - Badge/poster templates can have additional images besides the background image
   (:pr:`5273`, thanks :user:`SegiNyn`)
+- Add ability to add alerts to iCal exports (:issue:`5318`, :pr:`5320`, thanks
+  :user: `PerilousApricot`)
```

Comment:
```suggestion
  :user:`PerilousApricot`)
```

not sure how that spaces ended up there (in case it was part of my original suggestion..) 👾